index.html

```diff
@@ -0,0 +1,78 @@
+    <div class="main-choose"><a href="#welcome-section">About</a></div>
+    <div class="main-choose"><a href="#projects">Work</a></div>
+    <div class="main-choose"><a href="#footer">Contact</a></div>
+  </nav>
+
+  <div id="welcome-section">
+    <h1 class="h1-welcome-section">Hey I am Mimic</h1>
+    <p class="p-welcome-section"> a web developer</p>
+  </div>
+
+  <div id="projects">
+    <h1 class="h1-projects">These are some of my projects</h1>
+    <hr size="5px" color="#fff" width="50%" style="margin-left: 25%;">
+    <div id="projects-div-one">
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+    </div>
+    <div id="projects-div-two">
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+    </div>
+  </div>
+
+
+  <div id="navbar">
+    <h1 class="h1-navbar">
+      Let's work together...
+    </h1>
+    <p class="p-navbar">How do you take your coffee?</p>
+
+    <div id="links">
+      <a class="social-media" href="https://www.facebook.com">FaceBook</a>
+      <a class="social-media" href="https://www.github.com">GitHub</a>
+      <a class="social-media" href="https://www.twitter.com">Twitter</a>
+      <a class="social-media" href="email@gmail.com">Send an email</a>
+      <a class="social-media" href="+2123456789">Call me</a>
+    </div>
+    <hr size="5px" color="#be3144">
+    <div id="footer">
+      <p class="footer-content">**This is just a fake portfolio. All the projects and contact details given are not
+        real.</p>
+      <p class="footer-content" style="margin-left: 200px;">© Created for Microverse</p>
+    </div>
```

Comment:
- [ ] Please use the `<footer>` tag for the footer content instead of `<div>.` We should always use semantic tags as we have gone through the lesson it's extremely important. Can you do this for us? 😄